File: p.js
// var start = new Date();
var pc = 0;
function isPrime(number)
                {
                    for (var a = 2; a < number; a++)
                    {
                        if (number % a == 0)
                        {
                            return false;
                        }
                    }
                    return true;
                }
for (var a = 2; a < 2000; a++)
                {
                    if (isPrime(a)){
                        pc++;
                      // process.stdout.write('X');
                    }
                }
// var end = new Date() - start;
// console.info("\n"+pc+"\nExecution time: %dms", end);

File: p.py
# import time
# start_time = time.time()
pc=0
def isPrime(number):
	for x in range(2,number - 1):
		if (number % x == 0):
			return False
	return True
for a in range(2,2000):
	if(isPrime(a)):
		pc=pc+1
# 		print("X")
# print("")
# print(pc,"--- %s seconds ---" % ((time.time() - start_time)*1000))